### PR TITLE
Update copyright headers for release 1.0

### PR DESCRIPTION
**Pull request by mpreiner**
This PR will update all copyright headers before we make release 1.0.

Note: Headers will be updated once all cvc5 1.0 PRs are in.

### CODE REVIEW

File: src/context/cdhashmap.h [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /******************************************************************************
  * Top contributors (to current version):
- *   Morgan Deters, Tim King, Dejan Jovanovic
+ *   Morgan Deters, MikolasJanota, Tim King
```
Comment (aniemetz) [inline src/context/cdhashmap.h:3]: space

File: src/expr/node_self_iterator.h [hunk 2]
```diff
@@ -1,10 +1,10 @@
 /******************************************************************************
  * Top contributors (to current version):
- *   Morgan Deters, Mathias Preiner, Andres Noetzli
+ *   Morgan Deters, Ouyancheng, Mathias Preiner
```
Comment (aniemetz) [inline src/expr/node_self_iterator.h:3]: `Ouyancheng` needs to be fixed, too

File: src/proof/dot/dot_printer.h [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /******************************************************************************
  * Top contributors (to current version):
- *   Diego Della Rocca de Camargos
+ *   Diego Della Rocca de Camargos, Haniel Barbosa, vinciusb
```
Comment (aniemetz) [inline src/proof/dot/dot_printer.h:3]: `vinciusb` needs to be fixed

Comment (HanielB) [reply src/proof/dot/dot_printer.h:3]: Should be "Vinicius Braga"

Comment (mpreiner) [reply src/proof/dot/dot_printer.h]: Not "Vinícius Braga Freire"?

**Review by aniemetz — APPROVED** [review]

**Pull request merged**